File: Armstrong_fun.cpp
#include <iostream>
using namespace std;
// method
void armstrong(int fin){  //void is return type.
    int match = fin;
    int r ;
    int arms = 0;
    while(fin>0){
        r = fin % 10;
        arms +=(r * r * r);
        fin=fin/10;
    }
    if( match == arms){
        cout<<"Armstrong number";
    }
    else{
        cout<<"not a Armstrong number";
    }

}

// main function 
int main() {
  
    int n;

    cout<<"enter the nummber for finding the amstrong ";
    cin>>n;
    armstrong(n);
   
    return 0;
}

File: Armstrong_fun.java
import java.util.Scanner;

public class Armstrong {
    
    public static void armstrong(int fin) {
        int original = fin;
        int arms = 0;
        while (fin > 0) {
            int digit = fin % 10;
            arms += digit * digit * digit;
            fin /= 10;
        }
        if (original == arms) {
            System.out.println("Armstrong number");
        } else {
            System.out.println("Not an Armstrong number");
        }
    }
    
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        System.out.println("Enter a number to check if it's an Armstrong number: ");
        int fin = scanner.nextInt();
        armstrong(fin);
        
        scanner.close();
    }
}
